Bio/ExPASy/cellosaurus.py

```diff
@@ -87,9 +87,9 @@ class Record(dict):
 
     Each record contains the following keys:
 
-     ---------  ---------------------------     ----------------------
+     ---------  ------------------------------  -----------------------
```

Comment:
This table isn't quite right for a simple RST table:

https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#simple-tables

You can see how the old version didn't work either:

https://biopython.org/docs/1.83/api/Bio.ExPASy.cellosaurus.html

Can you change the minus signs into equals signs, and add a matching line after the final row? i.e. it should look like this (indented):

```rst
=========  ==============================  =======================
Line code  Content                         Occurrence in an entry
=========  ==============================  =======================
ID         Identifier (cell line name)     Once; starts an entry
AC         Accession (CVCL_xxxx)           Once
AS         Secondary accession number(s)   Optional; once
SY         Synonyms                        Optional; once
DR         Cross-references                Optional; once or more
RX         References identifiers          Optional: once or more
WW         Web pages                       Optional; once or more
CC         Comments                        Optional; once or more
ST         STR profile data                Optional; twice or more
DI         Diseases                        Optional; once or more
OX         Species of origin               Once or more
HI         Hierarchy                       Optional; once or more
OI         Originate from same individual  Optional; once or more
SX         Sex of cell                     Optional; once
AG         Age of donor at sampling        Optional; once
CA         Category                        Once
DT         Date (entry history)            Once
//         Terminator                      Once; ends an entry
=========  ==============================  =======================
```

(You can have a single column of spaces rather than two, but that's more work for little gain)

Reply:
Done!